aspnetcore/tutorials/grpc/grpc-start.md

```diff
@@ -3,15 +3,335 @@ title: Create a .NET Core gRPC client and server in ASP.NET Core
 author: juntaoluo
 description: This tutorial shows how to create a gRPC Service and gRPC client on ASP.NET Core. Learn how to create a gRPC Service project, edit a proto file, and add a duplex streaming call.
 ms.author: johluo
-ms.date: 08/20/2021
+ms.date: 09/13/2021
 no-loc: [Home, Privacy, Kestrel, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
 uid: tutorials/grpc/grpc-start
 ---
 # Tutorial: Create a gRPC client and server in ASP.NET Core
 
 By [John Luo](https://github.com/juntaoluo)
 
-::: moniker range=">= aspnetcore-5.0"
+::: moniker range=">= aspnetcore-6.0"
+This tutorial shows how to create a .NET Core [gRPC](xref:grpc/index) client and an ASP.NET Core gRPC Server.
+
+At the end, you'll have a gRPC client that communicates with the gRPC Greeter service.
+
+[View or download sample code](https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/tutorials/grpc/grpc-start/sample6) ([how to download](xref:index#how-to-download-a-sample)).
+
+In this tutorial, you:
+
+> [!div class="checklist"]
+> * Create a gRPC Server.
+> * Create a gRPC client.
+> * Test the gRPC client service with the gRPC Greeter service.
+
+## Prerequisites
+
+# [Visual Studio](#tab/visual-studio)
+
+[!INCLUDE[](~/includes/net-prereqs-vs-6.0.md)]
+
+# [Visual Studio Code](#tab/visual-studio-code)
+
+[!INCLUDE[](~/includes/net-prereqs-vsc-6.0.md)]
+
+# [Visual Studio for Mac](#tab/visual-studio-mac)
+
+We hope to have Visual Studio for Mac instructions soon.
+<!--
+[!INCLUDE[](~/includes/net-prereqs-mac-6.0.md)]
+-->
+
+---
+
+## Create a gRPC service
+
+# [Visual Studio](#tab/visual-studio)
+
+* Start Visual Studio 2022 and select **Create a new project**.
+* In the **Create a new project** dialog, select **ASP.NET Core gRPC Service** and select **Next**.
```

Comment:
I think that's to hard to find. Better tell them to enter **gRPC** in the search box.
MIght need a image too.